<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigDialog</class>
 <widget class="QDialog" name="ConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>设置</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>511</width>
     <height>401</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="tabsClosable">
    <bool>false</bool>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="tabBarAutoHide">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>文件</string>
    </attribute>
    <widget class="QLineEdit" name="trainExportLine">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>371</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>车次信息导出路径:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="changeFilePathButton">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>33</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>浏览...</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>车票信息导出路径:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>车次信息导入路径:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ticketExportLine">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>371</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="trainImportLine">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>371</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="ticketImportLine">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>260</y>
       <width>371</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>241</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>车票信息导入路径:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ticketExportButton">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>103</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>浏览...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="trainImportButton">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>173</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>浏览...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ticketImportButton">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>243</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>浏览...</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="exportCheckBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>320</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>启用自动导出</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="importCheckBox">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>320</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>启用自动导入</string>
     </property>
    </widget>
    <widget class="QPushButton" name="confirmButton">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>310</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
    <widget class="QPushButton" name="quitButton">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>310</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>用户</string>
    </attribute>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>管理员用户名:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>管理员密码:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="usernameLine">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>201</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="passwordLine">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>201</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="changeUserButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>89</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
    <widget class="QPushButton" name="userBackButton">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>170</y>
       <width>89</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
